src/session/ClientSession.h

```diff
@@ -23,16 +23,20 @@ struct SpaceInfo {
     meta::cpp2::SpaceDesc spaceDesc;
 };
 
-class Session final {
+class ClientSession final {
```

Comment:
Why rename?

Reply:
common has the session struct to save the session infomation.

Reply:
It's not conflict.

Reply:
It's not about conflict, it's about separation

Reply:
It's done by the namespace already.